### PR TITLE
Add extra class for card body

### PR DESCRIPTION
**Pull request by shankhadevpadam**
| Question                | Answer
| ----------------------- | -----------------------
| Issue or Enhancement    | Issue|Enhancement
| License                 | MIT

This is a pull request for help to add an extra class for the card body.  There is a problem before when using a responsive data table inside the card. There is no attribute to pass extra class for card body. 

Before

```php
<x-adminlte-card title="Card Title"></x-adminlte-card>
```

```html
<div class="card">
    <div class="card-body"></div>
</div>
```

After

```php
<x-adminlte-card title="Card Title" body-class="table-responsive"></x-adminlte-card>
```

```html
<div class="card">
    <div class="card-body table-responsive"></div>
</div>
```

#### Checklist

- [x] I tested these changes.
- [x] I have linked the related issues.

### CODE REVIEW

**Comment by Shidersz** [comment]
Thanks, I will review this soon...

**Review by Shidersz — APPROVED** [review]

**Comment by Shidersz** [comment]
@shankhadevpadam I have reviewed these changes and made some small fixes. The extra classes for the card body could be helpful in some particular situations. However, and just to note, in your particular case all the problem may be fixed if you define a wrapper `div` with the `table-responsive` class, for example:

```blade
<x-adminlte-card title="Card Title">
    <div class="table-responsive">
       {{-- Here goes the datatable markup... --}}
    </div>
</x-adminlte-card>
```

We now wait for @resslinger approval before merging this one...

**Review by resslinger — APPROVED** [review]

**Pull request merged**
